addons/binding/org.openhab.binding.magentatv/src/main/java/org/openhab/binding/magentatv/internal/network/MagentaTVNotifyServlet.java

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.magentatv.internal.network;
+
+import static org.openhab.binding.magentatv.internal.MagentaTVBindingConstants.*;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Scanner;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang.StringUtils;
+import org.openhab.binding.magentatv.internal.MagentaTVHandlerFactory;
+import org.openhab.binding.magentatv.internal.MagentaTVLogger;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+
+/**
+ * Main OSGi service and HTTP servlet for MagentaTV NOTIFY.
+ *
+ * @author Markus Michels (markus7017) - Initial contribution (derived from
+ *         Netatmo binding, thanks)
+ */
+@Component(service = HttpServlet.class, configurationPolicy = ConfigurationPolicy.OPTIONAL, immediate = true)
+public class MagentaTVNotifyServlet extends HttpServlet {
+    private static final long serialVersionUID = 2119809008606371618L;
+    private final MagentaTVLogger logger = new MagentaTVLogger(MagentaTVNotifyServlet.class, "NotifyServlet");
+
+    private HttpService httpService;
+    private MagentaTVHandlerFactory handlerFactory;
+
+    /**
```

Comment:
Comments for activate/deactive are not necessary (don't need to be changed, just a note)

Reply:
removed